### ISSUE TITLE
Invalid string error

### ISSUE THREAD

**Issue by luni3359** [b3nj5m1n/xdg-ninja#22]
The script runs flawlessly after switching the shebang to bash.

The error is caused by the read on line 173.
https://github.com/b3nj5m1n/xdg-ninja/blob/91a00011d2122a3f1ebb12e45b404373ff099c27/xdg-ninja.sh#L170-L176

![Terminal with "invalid string" errors](https://user-images.githubusercontent.com/16832398/168453253-85924d37-4ef1-4a66-b180-8c91286e4daa.png)

**Comment by b3nj5m1n** [comment]
That's pretty weird, what do you have _/bin/sh_ symlinked to?

**Comment by luni3359** [comment]
It's aliased to `/bin/dash`.

**Comment by b3nj5m1n** [comment]
I don't have any experience with that unfortunately, maybe @Ballasi could take a look at this as well?

**Comment by luni3359** [comment]
I'm using KDE Neon, which is based on Ubuntu. Please let me know if you want to know any more details.

**Comment by Ballasi** [comment]
This is happening due to the fact that `echo` behaves differently between `bash` and `dash`:
```
 $ cat /tmp/test.sh
#/bin/sh
echo "One\nTwo"
 $ bash /tmp/test.sh
One\nTwo
 $ dash /tmp/test.sh
One
Two
```
and from what I can read, jq is following JSON guidelines, which means forbidding multi-line strings with double quotes.

On `bash`, it works fine as the JSON given has a plain `\n`, whereas on dash, it has a literal newline, which breaks the program.

I'd say that there is no real reason to send the data from the plain JSON files to `stdout` when `jq` already can parse from a file via CLI. I published a fix in #23.
